compiler/rustc_driver/src/lib.rs

```diff
@@ -528,8 +528,13 @@ fn stderr_isatty() -> bool {
 }
 
 fn handle_explain(registry: Registry, code: &str, output: ErrorOutputType) {
-    let normalised =
-        if code.starts_with('E') { code.to_string() } else { format!("E{0:0>4}", code) };
+    let normalised = if code.starts_with('E') {
+        code.to_string()
+    } else if code.starts_with('e') {
```

Comment:
It's simpler to do `let normalized = code.to_ascii_uppercase();` before checking for prefix.

Reply:
Sure!